src/controllers/TermcrController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace macfly\taxonomy\controllers;
+
+use Yii;
+use yii\console\Controller;
+use macfly\taxonomy\models\Term;
+use yii\helpers\ArrayHelper;
+use yii\db\Expression;
+
+//cronjob to delete all ununsed term
+class TermcrController extends Controller {
+     public function actionIndex()
+     {
+       $expression = isset(Yii::$app->params['termPeriod']) ? 'DATE_SUB(NOW(), INTERVAL '.Yii::$app->params['termPeriod'].' DAY)' : 'DATE_SUB(NOW(), INTERVAL 30 DAY)';
```

Comment:
Why not just do : 
$delay = is_int(Yii::$app->params['termPeriod']) ? Yii::$app->params['termPeriod'] : 30;
$expression = 'DATE_SUB(NOW(), INTERVAL '. $delay .' DAY)';